časová-platnost/draft/index.html

```diff
@@ -310,7 +310,7 @@ <h3>Komplexní časová platnost</h3>
                    Příklad a schéma jsou stále ve vývoji.
                 </p>
 
-                <aside class="example" title="Jednoduché oznámení v JSON">
+                <aside class="example" title="Oznámení v JSON">
```

Comment:
Proč má Časová platnost v názvu příkladu Oznámení?

Reply:
@jakubklimek protože jsem bral, že tam nebude jednoduché a komplexní, tak jsem dal jen oznámení. To je špatně? díky za upřesnění

Reply:
@jirkamarekbrno to je v pohode, co je špatně je to, že to není žádné oznámení, viz komentář dole pro @spalrobert

Reply:
@jakubklimek oprava viz [https://github.com/jirkamarekbrno/otevrene-formalni-normy/commit/94ed4d2bcb38c63d150873850358b83380a1c683](url)